### PR TITLE
Attempt to make multicall similar to Solidity Implementation

### PR DESCRIPTION
**Pull request by dhruvkelawala**
- return block_number along with result_len and result

### CODE REVIEW

File: contracts/Multicall.cairo [hunk 1]
```diff
@@ -1,49 +1,32 @@
...
 
 @view
-func multicall{
-        syscall_ptr: felt*,
-        range_check_ptr
-    } (
-        calls_len: felt,
-        calls: felt*
-    ) -> (result_len: felt, result: felt*):
+func aggregate{syscall_ptr : felt*, range_check_ptr}(calls_len : felt, calls : felt*) -> (
```
Comment (juniset) [inline contracts/Multicall.cairo:16]: Can you keep the existing formatting to be consistent with the other contracts of the repo?

**Review by juniset — CHANGES_REQUESTED** [review]
Thanks for your contribution!

To move ahead with this PR please make sure to update the tests in `test/multicall.py`.

**Comment by juniset** [comment]
Also, I've just created the `develop` branch; could you target your PR to that branch?

**Comment by juniset** [comment]
Closed by #29

**Comment by dhruvkelawala** [comment]
Sorry I was busy with Jediswap. Good to see that you have included my changes :)

**Pull request closed without merging**